tests/meson.build

```diff
@@ -385,6 +385,7 @@ foreach backend : backends
     'vertical_rl',
     'vertical_lr',
     'fixed',
+    'transformed',
```

Comment:
- transformed -> animated
- tests: add a 3D animated text test

Reply:
I don't really mind changing it but what's your reasoning here? I chose "transform" because it's about the 3D space transformation of the geometry box, while "animated" could be really anything (like, letters being animated or the text bouncing around).

Reply:
We use `animated` almost everywhere else.

Reply:
okay, done